tasks/request_files/test/unit_tests/test_request_files.py

```diff
@@ -795,14 +839,13 @@ def test_inner_task_error_posting_status_raises(
         retry_sleep_secs = randint(0, 99)  # nosec
         recovery_type = uuid.uuid4().__str__()
         restore_expire_days = randint(0, 99)  # nosec
-        mock_s3_cli = mock_boto3_client("s3")
 
         mock_get_s3_object_information.return_value = {"StorageClass": "DEEP_ARCHIVE"}
 
         mock_create_status_for_job.side_effect = Exception("mock insert failed error")
 
         with self.assertRaises(Exception) as cm:
-            result = request_files.inner_task(
+            result = request_files.inner_task(  # noqa
```

Comment:
What's it complaining about here?

Reply:
test/unit_tests/test_request_files.py:848:13: F841 local variable 'result' is assigned to but never used

Reply:
can probably remove `result=`